bin/live/pycbc_live_single_trigger_fits

```diff
@@ -60,6 +61,17 @@ parser.add_argument("--duration-bin-spacing",
                     help="How to set spacing for bank split "
                          "if using --duration-bin-start, --duration-bin-end "
                          "and --num-duration-bins.")
+parser.add_argument('--prune-loudest', type=int,
+                    help="Number of loudest triggers to remove from each "
+                         "bin (and all triggers within --prune-window "
+                         "seconds around it)")
+parser.add_argument("--prune-window", type=float, default=0.1,
+                    help="Window (seconds) either side of the --prune-loudest "
+                         "loudest triggers in each duration bin to remove. "
+                         "Default=0.1s")
+parser.add_argument("--minimum-prune-stat", type=float, default=0,
```

Comment:
I would actually put a default of 8. here, on the philosophy that for low latency the default should be at least close to what we want. @titodalcanton what do you think?

Reply:
I was 50-50 on whether to go for 8 or 0, so 8 is fine by me

Reply:
Do we need to have defaults here, or does it make sense to require these options to be explicit?

Also, I would rename this option `--prune-stat-threshold` to keep the same prefix as the related options.

Reply:
We don't _need_ any defaults, but it is up to you if we want to more follow a philosophy of 'have every setting visible in options' or 'stuff should just work with the defaults, save explicit options for things we will actually want to change'

Reply:
I think for live stuff, having setting visible in options is probably the best way to go, just so things are kept in config (in case the defaults ever change). I will remove